src/main/webapp/WEB-INF/jsp/tiles/views/home/species-summary.jsp

```diff
@@ -8,19 +8,60 @@
 
 <!-- Set to http://localhost:8080/gxa/ or http://localhost:8080/gxa_sc/ -- Remember the trailing slash! -->
 <script>
-  document.addEventListener("DOMContentLoaded", function(event) {
+  document.addEventListener('DOMContentLoaded', function(event) {
+    var sliderSettings = {
+      dots: true,
+      infinite: true,
+      speed: 500,
+      slidesToShow: 6,
+      slidesToScroll: 6,
+      adaptiveHeight: true,
+      autoplay: true,
+      autoplaySpeed: 2000,
+      responsive: [
+        {
+          breakpoint: 1024,
+          settings: {
+            slidesToShow: 3,
+            slidesToScroll: 3,
+            infinite: true,
+            dots: true
+          }
+        },
+        {
+          breakpoint: 600,
+          settings: {
+            slidesToShow: 2,
+            slidesToScroll: 2,
+            dots: true
+          }
+        },
+        {
+          breakpoint: 480,
+          settings: {
+            slidesToShow: 1,
+            slidesToScroll: 1,
+            dots: true
+          }
+        }
+      ]
+    };
+
     homepageSpeciesSummaryPanel.render({
-      host: '${pageContext.request.contextPath}/',
-      resource: 'json/species-summary',
-      responsiveCardsRowProps: {
-        className: 'row expanded small-up-2 medium-up-3 large-up-6',
-        cardContainerClassName: 'column'
+        host: '${pageContext.request.contextPath}/',
+        resource: 'json/species-summary?limit=100',
+        carouselCardsRowProps: {
+          className: 'row expanded small-up-2 medium-up-3 large-up-6',
+          cardContainerClassName: 'column',
```

Comment:
Check closely whether these two class names are necessary (I think `cardContainerClassName` is but `className` isn’t), since the carousel has responsive options.

Reply:
I did test to remove either of `cardContainerClassName` and `className`, but they are necessary to keep the style of tabs with a small number of cards (less than the number of slides). Those cards are not rendered by slick component. FYI, carousel display does not need either of the classnames.

Please check the screenshots below with commented out `cardContainerClassName` and `className` respectively.

<img width="1680" alt="Screenshot 2019-06-05 at 16 38 54" src="https://user-images.githubusercontent.com/33519183/58970305-156df680-87b1-11e9-8e92-af2e167d877a.png">
<img width="1680" alt="Screenshot 2019-06-05 at 16 39 38" src="https://user-images.githubusercontent.com/33519183/58970306-156df680-87b1-11e9-8d71-7437649e1cbc.png">

Reply:
Alirght, I hadn’t thought about that! :+1: